README.md

```diff
@@ -1,21 +1,36 @@
-# Zarf - Kubernetes Air Gap Buddy 
+# Zarf - DevSecOps for Air Gap Systems 
 
 <img align="right" alt="zarf logo" src=".images/zarf-logo.png"  height="256" />
 
-Zarf massively simplifies the setup & administration of kubernetes clusters "across the [air gap](https://en.wikipedia.org/wiki/Air_gap_(networking))".
+## Why is Zarf Needed In This World?
+
+Most of the software ecosystem assumes your systems have access to the internet.  The world (for good reasons) has become more and more dependent upon Software as a Serivice (SaaS), which assume a robust connection to the internet and a willingness to inheritently trust 3rd party providers. Although this makes sense for 99% of the world, there are certain SECURE systems that must operate either fully disconencted, semi-disconnected, or might just want the ability to disconnect in case of emergencies (like while under an active cyber attack). Although only a small percentage of systems, these SECURE systems make up some of the most vital systems in the world, such as Aerospace and Defense, Finance, Healthcare, Energy, Water and Sewage, as well as many Federal, Local, and State Goverment systems.  
+
+These SECURE systems need a way to continously and securely deliver software too. Zarf exists to make it easy for open source, COTS, and organic software solutions to be delivered to secure and disconnected systems. Because although such systems might be small in number, they represent many of the most important systems in the world.  
+
+## What is Zarf?
+
+Zarf massively simplifies the setup & administration of kubernetes clusters that support DevSecOps "across the [air gap](https://en.wikipedia.org/wiki/Air_gap_(networking))".
 
 It provides a static go binary CLI that can pull, package, and install all the things your clusters need to run.  It caches downloads (for speed), hashes packages (for security), and can even _install the kubernetes cluster itself_ if you want it to.
 
 Zarf runs on [a bunch of operating systems](./docs/supported-oses.md) and aims to support configurations ranging from "I want to run one, simple app" to "I need to support & dependency control a _bunch_ of internet-disconnected clusters".
 
-&nbsp;
+## Explain Zarf Like I'm Ten
```

Comment:
Do we want to modify this at all after collecting feedback over on linkedin? Or leave it?